be/src/cloud/cloud_meta_mgr.cpp

```diff
@@ -826,18 +826,7 @@ Status CloudMetaMgr::get_storage_vault_info(
                                   });
     }
     for (const auto& vault : resp.storage_vault()) {
-        THdfsParams params;
-        params.fs_name = vault.hdfs_info().build_conf().fs_name();
-        params.user = vault.hdfs_info().build_conf().user();
-        params.hdfs_kerberos_keytab = vault.hdfs_info().build_conf().hdfs_kerberos_keytab();
-        params.hdfs_kerberos_principal = vault.hdfs_info().build_conf().hdfs_kerberos_principal();
-        for (const auto& confs : vault.hdfs_info().build_conf().hdfs_confs()) {
-            THdfsConf conf;
-            conf.key = confs.key();
-            conf.value = confs.value();
-            params.hdfs_conf.emplace_back(std::move(conf));
-        }
-        vault_infos->emplace_back(vault.id(), std::move(params));
+        vault_infos->emplace_back(vault.id(), std::move(vault.hdfs_info()));
```

Comment:
```suggestion
        if (vault.has_hdfs_info()) {
            vault_infos->emplace_back(vault.id(), vault.hdfs_info());
        }
```
std::move has no effect on const